components/Players/PlayerManager.vue

```diff
@@ -312,6 +310,9 @@ export default Vue.extend({
       }
     });
   },
+  beforeMount() {
+    this.supportedFeatures = getSupportedFeatures();
+  },
```

Comment:
Is this done client/server side?

It needs to be done client.

Reply:
It seems done client-side here. At least it's not crashing like it was in created().

Ideally though, we should move this to a plugin and define it as client-side that way.